packages/slider/src/HandleController.ts

```diff
@@ -0,0 +1,540 @@
+     * @param: PointerEvent on slider
+     * @return: Slider value that correlates to the position under the pointer
+     */
+    private calculateHandlePosition(
+        event: PointerEvent | MouseEvent,
+        model: ModelValue
+    ): number {
+        const rect = this.boundingClientRect;
+        const minOffset = rect.left;
+        const offset = event.clientX;
+        const size = rect.width;
+
+        const normalized = (offset - minOffset) / size;
+        const value = model.normalization.fromNormalized(
+            normalized,
+            model.range.min,
+            model.range.max
+        );
+
+        /* c8 ignore next */
+        return this.host.isLTR ? value : model.range.max - value;
+    }
+
+    public renderHandle(
+        model: ModelValue,
+        index: number,
+        zIndex: number
+    ): TemplateResult {
+        const isActive = this.activeHandle === model.name;
+        const classes = {
+            handle: true,
+            dragging: this.draggingHandle?.handleName === model.name,
+            'handle-highlight': model.highlight,
+        };
+        const style = {
+            [this.host.isLTR ? 'left' : 'right']: `${
+                model.normalizedValue * 100
+            }%`,
+            'z-index': zIndex.toString(),
+            // Allow setting background per-handle
+            'background-color': `var(--spectrum-slider-handle-background-color-${index}, var(--spectrum-slider-handle-default-background-color))`,
+            'border-color': `var(--spectrum-slider-handle-border-color-${index}, var(-spectrum-slider-handle-default-border-color))`,
+        };
+        return html`
+            <div
+                class=${classMap(classes)}
+                name=${model.name}
+                style=${styleMap(style)}
+                @manage=${streamingListener(
+                    {
+                        type: 'pointerdown',
+                        fn: (event) => this.handlePointerdown(event, model),
+                    },
+                    {
+                        type: 'pointermove',
+                        fn: (event) => this.handlePointermove(event, model),
+                    },
+                    {
+                        type: ['pointerup', 'pointercancel'],
+                        fn: (event) => this.handlePointerup(event, model),
+                    }
+                )}
+                role="presentation"
+            >
+                <input
+                    type="range"
+                    class="input"
+                    id=${ifDefined(isActive ? 'input' : undefined)}
+                    min=${model.clamp.min}
+                    max=${model.clamp.max}
+                    step=${model.step}
+                    value=${model.value}
+                    aria-disabled=${ifDefined(
+                        this.host.disabled ? 'true' : undefined
+                    )}
+                    aria-valuetext=${this.formattedValueForHandle(model)}
```

Comment:
That is a good question. Are there some accessibility folks that we ask for advice? I think we need to give some indication of which handle you are interacting with...

Reply:
@jnurthen thoughts on where we might back up our labeling approach here?

Reply:
@Westbrook have you looked at the react-spectrum RangeSlider? https://react-spectrum.adobe.com/react-spectrum/RangeSlider.html

Reply:
I have pushed a reworking of the labels based on the React Spectrum implementation. A basic summary is:
- the `<input>` for each handle is labelled by the slider's label and by itself
- there is an `aria-label` attribute on the `sp-slider-handle` that is passed to the `input`

See the range slider example on the slider-handle docs page to play with it.

The one difference is that React Spectrum automatically adds the `aria-labels` "Minimum" and "Maximum" to the inputs on the range slider. I have not done that since our solution is a little more free-form. But I could add that default for the two handle case if it made sense.